tests/pom.xml

```diff
@@ -23,6 +23,7 @@
 
         <!-- integration tests -->
         <module>ae-inventory-subjects-itest</module>
+        <module>ae-inventory-container-itest</module>
```

Comment:
Is this safe? Github actions and local procedures?

Reply:
idk about safeness, but since there is nothing too meaningful in tests, they cannot fail. only if we change something meaningful in the contributors and the collection and inventories change, but then we would also have to adjust the tests though

Reply:
ok

Reply:
Ich habe den PR aus dem Draft gehoben, wir werden sehen, ob Github den Branch bauen kann.